examples/jl-lab-extension-slate-steps.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
<p>We don't need these as python 2 is no longer supported. Joyce also did cleanup of these python2 imports from other notebooks a while ago.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Caltech-IPAC/firefly_client/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>